backend/src/utils/Constants.ts

```diff
@@ -2,10 +2,10 @@ import { IHttpError } from "../interfaces/IApiResponses";
 import { HTTPError } from "./Errors";
 
 export const envVars: string[] = [
-  "POSTGRESQL_HOST",
-  "POSTGRESQL_USER",
-  "POSTGRESQL_PASSWORD",
-  "POSTGRESQL_DATABASE",
+  // "POSTGRESQL_HOST",
+  // "POSTGRESQL_USER",
+  // "POSTGRESQL_PASSWORD",
+  // "POSTGRESQL_DATABASE",
```

Comment:
Should be these deleted?

Reply:
hm. good point for production, but for local dev run we wanna pass these in config files. probably will add it back and make the function only run for NODE_ENV==prod